pkg/securitycontextconstraints/sccadmission/scc_exec.go

```diff
@@ -61,7 +61,7 @@ func (d *sccExecRestrictions) Validate(a admission.Attributes, o admission.Objec
 
 	// TODO, if we want to actually limit who can use which service account, then we'll need to add logic here to make sure that
 	// we're allowed to use the SA the pod is using.  Otherwise, user-A creates pod and user-B (who can't use the SA) can exec into it.
-	createAttributes := admission.NewAttributesRecord(internalPod, nil, coreapi.Kind("Pod").WithVersion(""), a.GetNamespace(), a.GetName(), a.GetResource(), "", admission.Create, false, a.GetUserInfo())
+	createAttributes := admission.NewAttributesRecord(internalPod, nil, coreapi.Kind("Pod").WithVersion(""), a.GetNamespace(), a.GetName(), a.GetResource(), "", admission.Create, a.GetOperationOptions(), false, a.GetUserInfo())
```

Comment:
I think it is correct, based on https://github.com/kubernetes/kubernetes/pull/77563

Reply:
It wires through the field manager (of a pod) and dry run options. I tend to erase the dryrun options.

Reply:
Seems like it's better wire through than to snip them.

Reply:
passing ExecOptions (or what it is called) to a create admissoin is wrong. This is an artificial admission call.